wasm/src/account/address.rs

```diff
@@ -67,6 +71,20 @@ impl Address {
         self.0.to_string()
     }
 
+    /// Get the left endian byte array representation of the address plaintext.
+    #[wasm_bindgen(js_name = "toPlaintextBytesLe")]
+    pub fn to_plaintext_bytes_le(&self) -> Result<Uint8Array, String> {
+        let plaintext = Plaintext::from(LiteralNative::Address(self.0));
+        plaintext.to_bytes_le()
+    }
+
+    /// Get the left endian boolean array representation of the address plaintext bits.
+    #[wasm_bindgen(js_name = "toPlaintextBitsLe")]
+    pub fn to_plaintext_bits_le(&self) -> Array {
+        let plaintext = Plaintext::from(LiteralNative::Address(self.0));
+        plaintext.to_bits_le()
+    }
+
```

Comment:
Can you add `fromBytesLe`, `toBytesLe`, `fromBitsLe` `toBitsLe`, `toPlaintext`, `toFields`, `fromFields` on this?